nodejs/index.js

```diff
@@ -0,0 +1,92 @@
+import { createClient } from 'redis'
+
+const fnc = async() => {
+    // Lets connect
+    const client = createClient({ url:'redis://localhost:6379' });
```

Comment:
```suggestion
    const client = createClient({ url:'redis://localhost:6379' })
```
Just removing `;` for consistency with the rest of the script.